examples/react-native/app.json

```diff
@@ -0,0 +1,32 @@
+{
+  "expo": {
+    "name": "telefunc-demo",
+    "slug": "telefunc-demo",
+    "version": "0.0.1",
+    "orientation": "portrait",
+    "icon": "./assets/icon.png",
+    "splash": {
+      "image": "./assets/splash.png",
+      "resizeMode": "contain",
+      "backgroundColor": "#ffffff"
+    },
+    "updates": {
+      "fallbackToCacheTimeout": 0
+    },
+    "assetBundlePatterns": [
+      "**/*"
+    ],
```

Comment:
Are these lines required? Can we remove them?

Reply:
These are also added by default in a blank Expo project. I tried removing them and the project seems to be working fine without them.